### PR TITLE
RHDEVDOCS-4494: Added RN updates for Developer catalog or one of it's…

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Aligned team: Dev tools

OCP Version - 4.12

JIRA Link: https://issues.redhat.com/browse/RHDEVDOCS-4494

Preview page with content updates: https://52254--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-enabling-disabling-developer-catalog

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -100,6 +100,18 @@ In {product-title} 4.12, the following improvements have been made in the *Helm*
...
+==== Enabling or Disabling Developer catalog or one of it's sub-catalogs
+
+* With this update, the following options can be enabled or disabled in the Developer catalog by using either the form view or the YAML view:
+
+* Builder images
+* Devfiles
+* Helm Charts
+* Operator Backed
+* Samples
+* Templates
```
Comment (debsmita1) [inline release_notes/ocp-4-12-release-notes.adoc:113]: And a few more sub-catalog types show up in the developer catalog when certain operators are installed. For ex: 'Event Sinks' and 'Event Sources' show up when the Serverless operator is installed.  I think we should add those as well

Comment (Dhruv-Soni11) [reply release_notes/ocp-4-12-release-notes.adoc]: As discussed in Slack, I have included the additional options you suggested in the content (https://github.com/openshift/openshift-docs/pull/52218) and I am rewording the text a bit as per the suggestion of a peer in the doc team. Kindly view this change and provide your feedback, if any.

Comment (Dhruv-Soni11) [reply release_notes/ocp-4-12-release-notes.adoc]: Fixed.

**Comment by sanketpathak** [comment]
LGTM

**Review by christoph-jerolimov — APPROVED** [review]
Lgtm!

Content is fine, but its also included in #52342 (with less words). Is it fine that this is mentioned twice in the final RN?

**Comment by shipsing** [comment]
@Dhruv-Soni11, We have already discussed in our DPM call that this PR needs to be closed with reason mentioning duplicate issue. Could you please close it? we do not want to merge this content as this would result into duplicate content.

**Comment by Dhruv-Soni11** [comment]
> @Dhruv-Soni11, We have already discussed in our DPM call that this PR needs to be closed with reason mentioning duplicate issue. Could you please close it?

Yes @shipsing, as we discussed during the DPM call as the updates for this ticket are already being covered in your PR (https://github.com/openshift/openshift-docs/pull/52342), I will close this issue.

**Pull request closed without merging**